electricpy/__init__.py

```diff
@@ -1623,42 +1660,75 @@ def dynetz(delta=None,wye=None,round=None):
                 Wye-Connected impedance values { Z1, Z2, Z3 }
     """
     # Determine which set of impedances was provided
-    if(delta!=None and wye==None):
-        Z12, Z23, Z31 = delta # Gather particular impedances
-        Zsum = Z12 + Z23 + Z31 # Find Sum
+    if (delta != None and wye == None):
+        Z12, Z23, Z31 = delta  # Gather particular impedances
+        Zsum = Z12 + Z23 + Z31  # Find Sum
         # Calculate Wye Impedances
-        Z1 = Z12*Z31 / Zsum
-        Z2 = Z12*Z23 / Zsum
-        Z3 = Z23*Z31 / Zsum
-        Zset = ( Z1, Z2, Z3 )
-        if round!=None: Zset = _np.around(Zset,round)
-        return(Zset) # Return Wye Impedances
-    elif(delta==None and wye!=None):
-        Z1, Z2, Z3 = wye # Gather particular impedances
-        Zmultsum = Z1*Z2 + Z2*Z3 + Z3*Z1
+        Z1 = Z12 * Z31 / Zsum
+        Z2 = Z12 * Z23 / Zsum
+        Z3 = Z23 * Z31 / Zsum
+        Zset = (Z1, Z2, Z3)
+        if round != None: Zset = _np.around(Zset, round)
+        return (Zset)  # Return Wye Impedances
+    elif (delta == None and wye != None):
+        Z1, Z2, Z3 = wye  # Gather particular impedances
+        Zmultsum = Z1 * Z2 + Z2 * Z3 + Z3 * Z1
         Z23 = Zmultsum / Z1
         Z31 = Zmultsum / Z2
         Z12 = Zmultsum / Z3
-        Zset = ( Z12, Z23, Z31 )
-        if round!=None: Zset = _np.around(Zset,round)
-        return(Zset) # Return Delta Impedances
+        Zset = (Z12, Z23, Z31)
+        if round != None: Zset = _np.around(Zset, round)
+        return (Zset)  # Return Delta Impedances
     else:
         raise ValueError("ERROR: Either delta or wye impedances must be specified.")
 
-# Define Single Line Power Flow Calculator
-def powerflow( Vsend, Vrec, Zline ):
+#calculating impedance of bridge network
+def bridge_impedance(z1, z2, z3, z4, z5):
     """
-    Simple Power-Flow Calculator
+    bridge impedance calculator
```

Comment:
Forgive my pickiness here, but please consider capitalizing the phrase as: "Bridge Impedance Calculator" for consistency.